index.html

```diff
@@ -250,6 +250,17 @@
           </ul>
         </section>
 
+        <section id="drew" class="nametag">
+          <h4><a href="https://github.com/zarvox">Drew Fisher</a></h4>
+          <ul class="credentials">
+            <li>Software Engineer, <a href="https://www.aerofs.com/">AeroFS</a> 2012-2015
```

Comment:
I think you should mention your MS in usable security, probably as the first point. :)